test_regression.py

```diff
@@ -142,3 +152,69 @@ def test_mask_scl(auth_connection, tmp_path):
 
     # Excecute and assert
     execute_and_assert(cube, output_path, scenario_name)
+
+
+def test_BAP(auth_connection, tmp_path):
+
+    # Define scenario parameters
+    scenario_name = 'BAP'
+
+    # Set up output directory and path
+    output_path = tmp_path / f'output.nc'
+
+    # Parameters for data collection
+    collection_id = "SENTINEL2_L2A"
+    spatial_geometries = gpd.read_file('.\geofiles\BAP.geojson')
+    temporal_extent = ["2022-07-01", "2022-07-31"]
+    spatial_resolution = 20
+
+    # Fetch 100km x 100 km area from geojson
+    spatial_geometries = spatial_geometries.to_crs(epsg=4326)
+    area = eval(spatial_geometries.to_json())
+
+    # Get the spectral bands of interest
+    cube = auth_connection.load_collection(
+        collection_id,
+        temporal_extent = temporal_extent,
+        bands = ["B02", "B03","B04"],
+        max_cloud_cover=70
```

Comment:
Is there a reason you apply a different max_cloud_cover for the RGB bands than for the SCL band?

Reply:
I believe it is done so in the community example, will standardize